File: JavaScript/Arithmetic.js
class Arithmetic {
  constructor(a, b) {
    this.a = a;
    this.b = b;
  }

  get add() {
    return this.a + this.b;
  }
  get minus() {
    return this.a - this.b;
  }
  get multi() {
    return this.a * this.b;
  }
  get division() {
    return this.a / this.b;
  }
}

const testOne = new Arithmetic(4, 2).add;
const testTwo = new Arithmetic(4, 2).minus;
const testThree = new Arithmetic(4, 2).multi;
const testFour = new Arithmetic(4, 2).division;

console.log("더하기", testOne);
console.log("빼기", testTwo);
console.log("곱하기", testThree);
console.log("나누기", testFour);

File: Java/Arithmetic.java
public class Arithmetic {
  public static int add (int a, int b){
    return a + b;
  }
  public static int minus (int a, int b){
    return a - b;
  }
  public static int multi (int a, int b){
    return a * b;
  }
  public static int division (int a, int b){
    return a / b;
  }

  public static void main(String[] A){
    int a = 4;
    int b = 2;

    int testAdd = add(a, b);
    int testMinus = minus(a, b);
    int testMulti = multi(a, b);
    int testDivision = division(a, b);

    System.out.print(testAdd);
    System.out.print(testMinus);
    System.out.print(testMulti);
    System.out.print(testDivision);
  }
}